types/node-vault-client/index.d.ts

```diff
@@ -0,0 +1,273 @@
+declare class VaultClient {
+    /**
+     * Boot an instance of Vault
+     *
+     * The instance will be stored in a local hash. Calling Vault.boot multiple
+     * times with the same name will return the same instance.
+     *
+     * @param {String} name - Vault instance name
+     * @param {Object} [options] - options for {@link Vault#constructor}.
+     * @return Vault
+     */
+    static boot(name: string, options?: any): any;
```

Comment:
Isn't this `options` the same opts as the constructor, per the doc above? Such that you should pull out the options type and use it in both?

Reply:
Hi @jakebailey - Thank you for taking the time to review this PR. The following changes have been made:
- Remove unneeded descriptions from multiple Vault classes
- Enhance simplicity of constructor signatures across `VaultAppRoleAuth`, `VaultIAMAuth`, `VaultKubernetesAuth`, and `VaultTokenAuth`.